client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -72,6 +74,10 @@ class SiteTools extends Component {
 			'Delete all your posts, pages, media and data, ' +
 			'and give up your site\'s address.'
 		);
+		const manageConnectionTitle = translate( 'Manage your connection' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Manage Connection' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/site-settings-general-jetpack-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).